### PR TITLE
Completed task A for Polish language

### PR DESCRIPTION
**Pull request by MilyMilo**
Completed task A in separate branch. I've already claimed task on GCI so I'm waiting for approval. This is separate branch, but somehow commits from yesterday are listed here. This is only task A, commits from yesterday are deleted.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@MilyMilo, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by TomaszKolek** [comment]
Cześć @MilyMilo! Please do the interactive rebase and remove first 2 commits that are actually part of another task.

https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md - it should help

File: translations/Polish/research/notes.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+General notes:
+
+* The main diversity between Polish and English writing are the special letters, in English, there is no ą, ć, ę, ł, ń, ś, ó, ż, ź.
```
Comment (TomaszKolek) [inline translations/Polish/research/notes.md]: diversity? `difference sounds better IMHO

**Comment by TomaszKolek** [comment]
@MilyMilo Basically it looks really good! @trueskawka it'd be so nice if you could take a look at it as well.

Thanks Miłosz!

**Comment by MilyMilo** [comment]
Meh, sorry for causing problems :|  I'll fix few mistakes I caught while rebasing commits and request pull again.

**Comment by TomaszKolek** [comment]
@MilyMilo after creating new one please point here link to it

**Comment by MilyMilo** [comment]
Here it is: https://github.com/zulip/zulip-gci/pull/569

**Pull request closed without merging**